spec/spec_helper.rb

```diff
@@ -19,11 +19,26 @@
 
 # Code coverage analysis, output to coverage/ dir
 require 'simplecov'
+require 'webmock/rspec'
+
+WebMock.disable_net_connect!(allow_localhost: true)
 
 # Start SimpleCov
 SimpleCov.start
 
 RSpec.configure do |config|
+  
+  
+  config.before(:each) do
+    stub_request(:get, /127.0.0.1:52725/).
+      with(headers: {'Accept'=>'*/*', 'User-Agent'=>'Ruby'}).
```

Comment:
Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.